### DISCUSSION TITLE
Inlay hint error after update lazyvim

### DISCUSSION THREAD

**Discussion by Jiyangson** [LazyVim/LazyVim#3081] [Q&A] [answered]
I got the following error after I updated lazyvim to 10.22.0.
![image](https://github.com/LazyVim/LazyVim/assets/31755447/57edd0ec-2b36-4b8c-9f30-23fbf504a17b)
My neovim version is 0.10.0\~ubuntu1+git202401272103-2cd76a758-971e32c87-6850187d2\~ubuntu20.04.1, which I believe is the newest version in ubuntu 20.04.1.
I do not understand the reason completely but I guess it should be related to the recent change to toggle.lua
![image](https://github.com/LazyVim/LazyVim/assets/31755447/772602f0-a5de-4b49-ad16-aa93ab7f90f8)
I use command `:lua vim.lsp.inlay_hint.enable(0, not vim.lsp.inlay_hint.is_enable())` to enable inlay hint and it works fine. But how to automatically enable inlay hints for all buffers after startup? Thanks

**Comment by dpetka2001** [comment, marked as answer]
You have an older version of Neovim which was built on 27/01/2024. Upgrade Neovim to latest nightly. I use [bob](https://github.com/MordechaiHadad/bob) to manage Neovim installations and lets you easily switch between stable and nightly versions.

The recent change in `toggle` was made because in Neovim nightly they changed the signature of `inlay_hints`. If you want to use nightly you should be following always latest build because there are quite a few breaking changes in API. Otherwise, you might want to consider downgrading to stable.

**Reply by Jiyangson** [reply]
Thanks for your answer. This version is the newest version for ubuntu 20.04. To upgrade to latest nightly, I have to update my ubuntu system, which I will do at some point. For now I just bare with the warnings.

**Reply by dpetka2001** [reply]
As i've already said, you can use [bob](https://github.com/MordechaiHadad/bob) to install Neovim. It allows you to switch between Neovim stable and nightly and it doesn't depend on your OS package manager.
